### PR TITLE
Tic-tac-toe Game

### LINKED ISSUE #262: Tic-Tac-Toe Game in Python language
Tic-Tac-Toe Game in Python language

This is a console based two player game, entirely written in python language.

**Languages**

-   [ ] Python

### PR DESCRIPTION
**Pull request by arpitsomani8**
Python Game

Amazing game made with python.

Fixed #262

### CODE REVIEW

**Comment by kaustubhgupta** [comment]
@arpitsomani8 may I ask which issue this PR resolves?

**Comment by arpitsomani8** [comment]
The PR constitute and resolves issue of "python language program"

Tic-Tac-Toe
It is a famous and known game and I have implemented this entirely in
python language.

It is a 2 player game...

**Comment by kaustubhgupta** [comment]
Is there any issue raised in this Repository related to this which is open?

**Comment by arpitsomani8** [comment]
I am new to this, i will surely raise a issue first, before any PR.

**Comment by kaustubhgupta** [comment]
> I am new to this, i will surely raise a issue first, before any PR.

Being new is absolutely fine, Hacktoberfest is all about promoting open source contribution among beginners. Now please have a look at contribution guidelines to get started and also look at pinned issue "Important announcement"

File: Python/Tic-Tac-Toe/Tic-tac-toe.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+"""
+@author: Arpit Somani
+"""
+
+import numpy
```
Comment (kaustubhgupta) [inline Python/Tic-Tac-Toe/Tic-tac-toe.py:7]: - add comments to this program
- add a GIF to demonstrate/demo of the game
- place the GIF in this Tic-Tac-Toe folder
- Edit the readme of the python folder

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Python/Tic-Tac-Toe/Readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<h3>Concept Behind It :</h3>
```
Comment (kaustubhgupta) [inline Python/Tic-Tac-Toe/Readme.md]: This is not what I told you. I told you to update the Python folder readme with your entry. Kindly delete this file

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Python/Tic-Tac-Toe/Readme.md (hunk 2 above)
Comment (arpitsomani8) [reply Python/Tic-Tac-Toe/Readme.md]: ok

**Review by kaustubhgupta — APPROVED** [review]

**Pull request merged**